### DISCUSSION TITLE
React + tailwind

### DISCUSSION THREAD

**Discussion by fvlly** [tailwindlabs/tailwindcss#7238] [Help]
Hello, I followed the installation process, however, tailwind doesn't work in my react app, thanks in advance
![image](https://user-images.githubusercontent.com/60034721/151405858-c76f94bb-9776-4b5b-9f6e-18f8b2e2d031.png)

**Comment by kncs** [comment]
Hello @fvlly, you should try with the class "text-blue-400". "text-blue" is not a proper tailwind class.

See the documentation for more information 
https://tailwindcss.com/docs/text-color

Hope it helps :)

**Reply by fvlly** [reply]
@kncs Many thanks, it worked, albeit I get the following problem on vsCode "unknown at rule @tailwind", do have a nice day, thanks again.

**Reply by kncs** [reply]
@fvlly you are welcome

**Comment by Pearly-c** [comment]
Hello
So i configured a react environment for the first time, I'm not sure how to add tailwind to it...do i have to follow the same procedure for CREATE REACT APP? or is there a different procedure for that?

**Reply by kncs** [reply]
Hello @Pearly-c, if it's your first time with react env, I suggest you to create your app with CRA (Create React App) 
https://reactjs.org/docs/create-a-new-react-app.html#create-react-app

and then follow the procedure dedicated on the tailwind docs.
https://tailwindcss.com/docs/guides/create-react-app

Have fun with it :)

**Comment by fvlly** [comment]
@kncs Hello again, sorry for the bother, I tried creating a dynamic react+ tailwind component using classnames, however, some properties aren't reflecting (bg-color and text-color), could you enlighten me, thanks much 
![image](https://user-images.githubusercontent.com/60034721/151781799-84e6ce83-054c-47dd-9965-b465a5d64a31.png)

**Reply by kncs** [reply]
@fvlly, it's perfectly normal, tailwind will not parse or execute your code, so it will not add classes into the css built.
You will find full explanation on this link 

https://tailwindcss.com/docs/content-configuration#class-detection-in-depth

Thanks to tailwind for the goodness of the documentation.
